testing/runner.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env deno -A
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { parse } from "../flags/mod.ts";
+import { glob, walk } from "../fs/mod.ts";
+import { runTests } from "./mod.ts";
+const { args } = Deno;
+
+
+const { cwd } = Deno;
+const DEFAULT_GLOB = "**/*_test.ts";
```

Comment:
Is it possible to set an arg for it? Some people uses `.spec.ts` syntax for example

Reply:
Yeah, yeah, it will be controllable by flag. I'm just waiting for #556 to land and wrap up this PR

Reply:
Should also test test.ts